examples/ephemeral-spark/README.md

```diff
@@ -7,6 +7,7 @@ This example demonstrates a terraform-generated Tamr config for a full aws-scale
 - RDS Postgres instance
 - Tamr VM deployment
```

Comment:
It could be a good idea to add a brief description of what is the goal of this example.

Reply:
It is in [there](https://github.com/Datatamer/terraform-aws-tamr-config/blob/e6a4f0c16b495041ae2efda307c19f45759cf147/examples/ephemeral-spark/README.md)

Reply:
Fixed [here](f8d63edbb9bc7d586e2868759543046b8271a9a2)